src/main/java/org/primeframework/email/service/JavaMailEmailTransportService.java

```diff
@@ -53,17 +52,33 @@
  * @author Brian Pontarelli
  */
 public class JavaMailEmailTransportService implements EmailTransportService {
-  private static final ExecutorService ExecutorService;
+  private static final ExecutorService Executor;
```

Comment:
why is this final variable capitalized?

Reply:
Because it is `static` - traditional convention would be `EXECUTOR` but it feels like the code is yelling at me. :-)

Reply:
But now that I think about.. going to make this non static and just make this service a singleton.

Reply:
I've always used the Google Java style guide where only constants are in all caps.  https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Going with a singleton avoids this debate.  :)
looks good.

Reply:
yeah, I'm an idiot - my brain cross wires with static and constant. 🫤